src/utils/core.h

```diff
@@ -44,10 +44,20 @@ std::unordered_set<K> keySet(const std::unordered_map<K, V>& input) {
  * @return A string representation of that value, as a std::string. The exact representation of
  * the value is up to the implementation.
  */
-template <typename T> std::string to_string(const T& val) {
+template <typename T>
+std::string to_string(const T& val) {
 	return std::to_string(val);
 }
 
+/**
+ * @brief Converts a boolean to a string.
+ *
+ * @param val The value to get the string representation of.
+ * @return "true" iff val, otherwise "false".
```

Comment:
typo: `iff` -> `if`

Reply:
Not a typo, [iff](https://en.wikipedia.org/wiki/If_and_only_if) is short for "if and only if", which is pretty standard notation since it's taught in the early 300-level CSE courses. It's a mathematically stronger statement than "if", so imo it's preferrable when applicable

Reply:
Good to know.